### DISCUSSION TITLE
Use this package with  @next/bundle-analyzer ?

### DISCUSSION THREAD

**Discussion by ghost** [vinissimus/next-translate#874] [Q&A] [answered]
I am successfully using this package with nextjs, everything works. But now i like to analyse the bundle sizes and the recommended package "@next/bundle-analyzer" advises to wrap the config with their Object which at the otherhand obstructs this packge. Is there a way to use them both together in the same next.config.js file ??? Thanks

**Comment by miikebar** [comment]
Take a look at [next-compose-plugins](https://github.com/cyrilwanner/next-compose-plugins)

**Comment by aralroca** [comment, marked as answer]
Yes, we are using `@next/bundle-analyzer` with `next-translate` and `next-plugin-preact` without any problem.

```js
const hasBundleAnalyzer = withBundleAnalyzer({
  analyzeServer: ['server', 'both'].includes(process.env.BUNDLE_ANALYZE),
  analyzeBrowser: ['browser', 'both'].includes(process.env.BUNDLE_ANALYZE),
  bundleAnalyzerConfig: {
    server: {
      analyzerMode: 'static',
      reportFilename: 'server.html',
    },
    browser: {
      analyzerMode: 'static',
      reportFilename: 'client.html',
    },
  },
  ...config,
})

module.exports = nextTranslate(withPreact(hasBundleAnalyzer))
```

**Reply by ghost** [reply]
Got it running. Thanks a lot.
